<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sqlwindow</class>
 <widget class="QDialog" name="sqlwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #fff;
}
QPushButton {
    border: 2px solid gray;
    border-radius: 6px;
    min-width: 80px;
	font: 13pt &quot;Liberation Serif&quot;;
	color: black;
	background-color: white
}
QPushButton:pressed {
    border: 2px solid blue;
}
QComboBox {
    border: 2px solid gray;
    border-radius: 6px;
    min-width: 80px;
	font: 13pt &quot;Liberation Serif&quot;;
	color: black;
	background-color: white;
}
QLineEdit {
	color: black;
	background-color: #ddd;
 	border-radius: 8px;
 	border: 1px solid #e0e4e7;
	padding: 5px 15px;
}
QTableWidget {
	color: black;
}
QLabel {
	color: black;
	font: 11pt &quot;DejaVu Math TeX Gyre&quot;;
}
QMenu {
	color: black;
	border: 2px solid gray;
	border-radius: 12px 12px 0px 0px;
    min-width: 80px;
}
QMenuBar {
	color: black;
	border: 2px solid gray;
    min-width: 80px;
}</string>
  </property>
  <widget class="QLineEdit" name="sqlQuery">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="resultOfSelect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>571</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="resultOfQuery">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>561</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
